backend/crossref.py

```diff
@@ -405,7 +405,8 @@ def save_doi_metadata(self, metadata, extra_orcids=None):
         if metadata is None or type(metadata) != dict:
             raise ValueError('Invalid metadata format, expecting a dict')
         if not metadata.get('author'):
-            raise ValueError('No author provided')
+            # BareName has "last" as mandatory field
+            metadata['author'] = {'family': 'N/A'}
```

Comment:
This code is never going to be executed because it is just after an exception is raised.